dotnet/src/Connectors/Connectors.Amazon/Bedrock/Core/Models/AI21Labs/AI21JurassicRequest.cs

```diff
@@ -0,0 +1,203 @@
+        /// </summary>
+        [JsonPropertyName("applyToWhitespaces")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public bool? ApplyToWhitespaces { get; set; }
+
+        /// <summary>
+        /// Whether to apply penalty to punctuation.
+        /// </summary>
+        [JsonPropertyName("applyToPunctuations")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public bool? ApplyToPunctuations { get; set; }
+
+        /// <summary>
+        /// Whether to apply penalty to numbers.
+        /// </summary>
+        [JsonPropertyName("applyToNumbers")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public bool? ApplyToNumbers { get; set; }
+
+        /// <summary>
+        /// Whether to apply penalty to stop words.
+        /// </summary>
+        [JsonPropertyName("applyToStopwords")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public bool? ApplyToStopwords { get; set; }
+
+        /// <summary>
+        /// Whether to apply penalty to emojis.
+        /// </summary>
+        [JsonPropertyName("applyToEmojis")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public bool? ApplyToEmojis { get; set; }
+    }
+
+    /// <summary>
+    /// Fields that can be added to the penalty objects.
+    /// </summary>
+    [Serializable]
+    public class FrequencyPenalty
+    {
+        /// <summary>
+        /// Scale of the penalty.
+        /// </summary>
+        [JsonPropertyName("scale")]
+        public double Scale { get; set; }
+
+        /// <summary>
+        /// Whether to apply penalty to white spaces.
+        /// </summary>
+        [JsonPropertyName("applyToWhitespaces")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public bool? ApplyToWhitespaces { get; set; }
+
+        /// <summary>
+        /// Whether to apply penalty to punctuation.
+        /// </summary>
+        [JsonPropertyName("applyToPunctuations")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public bool? ApplyToPunctuations { get; set; }
+
+        /// <summary>
+        /// Whether to apply penalty to numbers.
+        /// </summary>
+        [JsonPropertyName("applyToNumbers")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public bool? ApplyToNumbers { get; set; }
+
+        /// <summary>
+        /// Whether to apply penalty to stop words.
+        /// </summary>
+        [JsonPropertyName("applyToStopwords")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public bool? ApplyToStopwords { get; set; }
+
+        /// <summary>
+        /// Whether to apply penalty to emojis.
+        /// </summary>
+        [JsonPropertyName("applyToEmojis")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public bool? ApplyToEmojis { get; set; }
+    }
```

Comment:
Since all 3 type of Penalty have exact same data structure, create a internal struct named `AI21RepetetionPenalty` (or chose an appropriate name), instead of defining 3 different type of Penalty inside Request class
```
internal struct AI21RepetetionPenalty
{
    /// <summary>
    /// Scale of the penalty.
    /// </summary>
    [JsonPropertyName("scale")]
    internal double Scale { get; set; }

    /// <summary>
    /// Whether to apply penalty to white spaces.
    /// </summary>
    [JsonPropertyName("applyToWhitespaces")]
    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
    internal bool? ApplyToWhitespaces { get; set; }

    /// <summary>
    /// Whether to apply penalty to punctuation.
    /// </summary>
    [JsonPropertyName("applyToPunctuations")]
    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
    internal bool? ApplyToPunctuations { get; set; }

    /// <summary>
    /// Whether to apply penalty to numbers.
    /// </summary>
    [JsonPropertyName("applyToNumbers")]
    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
    internal bool? ApplyToNumbers { get; set; }

    /// <summary>
    /// Whether to apply penalty to stop words.
    /// </summary>
    [JsonPropertyName("applyToStopwords")]
    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
    internal bool? ApplyToStopwords { get; set; }

    /// <summary>
    /// Whether to apply penalty to emojis.
    /// </summary>
    [JsonPropertyName("applyToEmojis")]
    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
    internal bool? ApplyToEmojis { get; set; }
}
```